website/docs/user-group-role/access-control/permissions.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "About permissions"
+description: "Learn about global and object permissions in authentik."
+---
+
+Permissions are the central components in all access control systems, the lowest-level component, the controlling piece of access data. Permissions are assigned to (or removed from!) to define exactly WHO can do WHAT to WHICH part of the overall software system.
+
+## Fundamentals of authentik permissions
+
+There are two main types of permissions in authentik:
+
+-   [**Global permissions**](#global-permissions)
+-   [**Object permissions**](#object-permissions)
+
+### Global permissions
+
+Global permissions define who can do what on a global level, across the entire system. Some examples in authentik are the ability to add new [flows](../../flow/index.md) or to create a URL for users to recover their login credentials.
+
+You can assign _global permissions_ to individual [users](../user/index.mdx) or to [roles](../roles/index.mdx) (most common and best practice).
```

Comment:
Seeking clarity: Which is the most common and best practice?  Or is assigning global permissions the best practice?

Reply:
Usually it is recommended to assign permissions to roles even if that role will only be used by a single user in a single group since it allows for easier adding of new users later, and it is usually also the most common

Reply:
I expanded the sentence to clarify which is the best practice.